### ISSUE TITLE
[BUGFIX] support for object_types

### ISSUE THREAD

**Issue by lhcstation** [valeoai/V-Max#19]
The bug is that, if i wana to enabling the object_types feat, the feature will be a jnp with shape (obj_num, 4, 1). When it comes to https://github.com/valeoai/V-Max/blob/b50f01f0c580e47214dbe6ee432d69d5ba2304da/vmax/simulator/features/features_datatypes.py#L48
it cannot concatenate, as other feature of ObjectFeatures has shape with (obj_num, time, ...), time is not equal with 4.

it is easy to fix, just broadcast the object_types with the time axis, in vec_extractor.py, https://github.com/valeoai/V-Max/blob/b50f01f0c580e47214dbe6ee432d69d5ba2304da/vmax/simulator/features/extractor/vec_extractor.py#L393
add:
```python
if key == "object_types":
    feature = jnp.broadcast_to(
        feature[:, None, :], 
        (
            self._num_closest_objects+1, 
            self._obs_past_num_steps, 
            4
        )
    )
```
